src/schemas/validation/schema.yaml

```diff
@@ -230,6 +230,24 @@ $defs:
         type: array
         items:
           $ref: '#/$defs/parameter-or-reference'
+      additionalOperations:
+        type: object
+        additionalProperties:
+          $ref: '#/$defs/operation'
+        propertyNames:
+          $comment: RFC9110 restricts methods to "1*tchar" in ABNF
+          pattern: "^[a-zA-Z0-9!#$%&'*+.-^`|~]+$"
+          not:
+            enum:
+              - GET
+              - PUT
+              - POST
+              - DELETE
+              - OPTIONS
+              - HEAD
+              - PATCH
+              - TRACE
+              - QUERY
```

Comment:
I agree this is a good form for this schema. I'm confused that these are upper case, though, when the property names of the `path-item` object are lower-case. Neither `path-item` nor `additionalOperations` can contain `GET` but both may contain `get`.

Reply:
@notEthan BTW this is part of why I argued that you _either_ used the existing fields _or_ used a new `operations` field that would handle all methods with the same casing rules (exact case match).  Because I think it's confusing to use a mixture of fields that are the "wrong" case for the method they produce plus a supplementary object that uses the exact case as-is.  But I lost that argument.

Reply:
Ah, this was said on #1747 which I didn't read or recall sufficiently. Thanks.